templates/components/itilobject/actors/field.html.twig

```diff
@@ -206,6 +206,7 @@
                return {
                   action: 'getActors',
                   actortype: actorytype,
+                  users_right: actorytype === 'assign' ? 'own_ticket' : 'all',
```

Comment:
I think  i'll prefer here to have just a right parameter here (and set it as 'all' if not defined.
And pass from it templates/components/itilobject/actors/main.html.twig when it is necessary (for the assign part)